src/services/TeamAssignmentService.ts

```diff
@@ -0,0 +1,49 @@
+import { PlayerInActiveMatch } from "../repositories/ActiveMatchRepository/types";
+import { PlayerInQueue } from "../repositories/QueueRepository/types";
+import { Team } from "../types/common";
+
+export async function createRandomTeams(
+  ballchasers: ReadonlyArray<PlayerInQueue>
+): Promise<Array<PlayerInActiveMatch>> {
+  const sortedBallChaser = ballchasers
+    .map((player) => {
+      return player;
+    })
+    .sort((o, b) => o.mmr - b.mmr);
+
+  const activeMatch: PlayerInActiveMatch[] = [];
```

Comment:
This is what is being returned to be passed to `addActiveMatch`, right? So shouldn't this type be `Array<NewActiveMatchInput>` instead of `Array<PlayerInActiveMatch>`?

Reply:
Yes sir.  Fixed in next commit.